KnockKnock-iOS/Scenes/Feed/FeedList/FeedListPresenter.swift

```diff
@@ -16,6 +16,7 @@ final class FeedListPresenter: FeedListPresenterProtocol {
   var view: FeedListViewController?
 
   func presentFetchFeedList(feedList: FeedList) {
+    LoadingIndicator.hideLoading()
     self.view?.fetchFeedList(feedList: feedList)
```

Comment:
```suggestion
    self.view?.fetchFeedList(feedList: feedList)
    LoadingIndicator.hideLoading()
```
[p2] 순서가 이게 맞지 않을까 싶네요?!

Reply:
데이터가 셋팅이 되고 로딩이 사라지는 구조로 말이죠!

Reply:
해당사항 반영하였습니다!